### DISCUSSION TITLE
テーブルマウンテンの完成度

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#80] [エリア・建築の完成度・開発状況]
- [x] 1～3F 地下水路
_土台、建築、装飾共にすべてクリア。※_
- [x] 4F 地下水路の休憩所
_土台、建築、装飾共にすべてクリア。※_
_仕掛けが未完成。_
- [ ] 5F～7F 溶岩アスレチック
_土台、建築、装飾共にすべてクリア。※_
- [x] 8F 幸福街 / 裏幸福街
_土台、建築、装飾共にすべてクリア。※_
_仕掛けが未完成。_
- [x] 9F～11F 木造迷宮
_土台、建築、装飾共にすべてクリア。※_
- [x] 12F～14F ZAP Zone
_土台、建築、装飾共にすべてクリア。※_
- [x] 15F 屋上
_土台、建築、装飾共にすべてクリア。※_

※ 鯖が1.16環境に移行した際、再度1.16ブロックを散りばめる予定。

**Comment by huyuyori** [comment]
![2020-11-19_16 55 40](https://user-images.githubusercontent.com/73624149/99639514-081ef780-2a8b-11eb-952b-550310b4bd81.png)
![2020-11-19_16 55 54](https://user-images.githubusercontent.com/73624149/99639518-09e8bb00-2a8b-11eb-9e21-b9c8e112c31c.png)
1.16化進行中

**Comment by huyuyori** [comment]
![2020-11-19_18 16 13](https://user-images.githubusercontent.com/73624149/99774855-affef880-2b51-11eb-808b-c83872decc7f.png)
ざっと1.16化完了 6～7Fが全域に装飾追加で変化あり 1～4F、8～10Fは1.16ブロック採用しつつも大きく変化なし 未完成の5Fと11F以降は一切変化なし

**Comment by anminmakura** [comment]
- [x] テーブルマウンテンの序盤の橋を新規製作
いい加減に古い橋を新規でどうにかしたい。

- [x] テーブルマウンテン初期地点を新規製作
同様に古い初期地点を新規で作り直したい。

**Comment by anminmakura** [comment]
ある程度橋は制作できたが、橋の装飾と橋の開始地点と

- [x] テーブルマウンテンの1Fにつながる終了地点の製作予定

- [x] また他の黄色黒の地点に灯台と土台の配置予定

**Comment by anminmakura** [comment]
テーブル5Fのランダムアスレ地帯の完成がまだ。
完成はるーく氏次第。
